### DISCUSSION TITLE
pagmo 2.19.0

### DISCUSSION THREAD

**Discussion by bluescarni** [esa/pagmo2#537] [General]
This new pagmo release features several fixes, a few feature additions and a performance optimisation for highly parallel workloads.

The full changelog, as usual, is available here:

https://esa.github.io/pagmo2/changelog.html

<hr /><em>This discussion was created from the release <a href='https://github.com/esa/pagmo2/releases/tag/v2.19.0'>pagmo 2.19.0</a>.</em>

**Comment by scudelari** [comment]
Hello!

Thank you so much for such an amazing work. 
Do you know when the new vcpkg package will be available?

Thank you!

**Reply by bluescarni** [reply]
Hi @scudelari ,

we do not maintain the vcpkg package, so your best bet would be to ask a vcpkg maintainer about it.

I have never made a vcpkg myself, but from what I have understood of their process chances are that a simple pull request to their recipe bumping up the pagmo version number might be enough.

**Reply by scudelari** [reply]
Hi Francesco,

I reached out to the package maintainer, Billy O'Neal <billy.oneal@gmail.com>. He has the following question. I confess I am not an expert in CMake, so could you perhaps help me to reply him?

Thanks!
------------------------
Hi there!

Sure, I submitted an update: https://github.com/microsoft/vcpkg/pull/29512/

I observe that our automatically generated usage is probably wrong:

pagmo2 provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(pagmo CONFIG REQUIRED)
    target_link_libraries(main PRIVATE Boost::boost Pagmo::pagmo Boost::disable_autolinking)

Can you confirm that a user should expect to do this?

pagmo2 provides CMake targets:

    find_package(pagmo CONFIG REQUIRED)
    target_link_libraries(main PRIVATE Pagmo::pagmo)

Thanks!
